src/methods.jl

```diff
@@ -164,40 +164,24 @@ One can also do weighted fits:
 
 See also `classes`, `support`.
 """
-function Dist.pdf(
-    d::UnivariateFinite{S,V,R,P},
-    cv::CategoricalValue,
-) where {S,V,R,P}
-    return get(d.prob_given_ref, int(cv), zero(P))
-end
-Dist.pdf(d::UnivariateFinite{S,V}, c::V) where {S,V} = _pdf(d, c)
-Dist.pdf(::UnivariateFinite{S,V}, ::Missing) where {S,V} = missing
-
-# Avoid method ambiguity errors with Dist >= 0.24
-Dist.pdf(d::UnivariateFinite{S,V}, c::V) where {S,V<:Real} = _pdf(d, c)
+Dist.pdf(::UnivariateFinite, ::Missing) = missing
 
-function _pdf(d::UnivariateFinite, c)
+function Dist.pdf(d::UnivariateFinite{S,V,R,P}, c) where {S,V,R,P}
```

Comment:
In this case do we still need `{S,V,R,P}`?

Reply:
yes, because `P` is used in line 173 below:
https://github.com/JuliaAI/CategoricalDistributions.jl/pull/40/files#diff-ee038c16114879c8c38f6b0986eec8fcdb079a629fbd6133e9875c85ae90c9dfR173